vars/template.go

```diff
@@ -264,6 +275,23 @@ func (t varsTracker) ExtraError() error {
 	return UnusedVarsError{Vars: names(unusedNames)}
 }
 
+var credTypeNameMatcher = regexp.MustCompile("creds\\.")
```

Comment:
What's this about?

Reply:
With my new implementation, this is no longer needed.